ReactiveCocoa/UIKit/UIActivityIndicatorView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIActivityIndicatorView.swift
+//  Rex
+//
+//  Created by Evgeny Kazakov on 02/07/16.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import ReactiveSwift
+import UIKit
+
+extension UIActivityIndicatorView {
+
+	/// Wraps an indicator's `isAnimating()` state in a bindable property.
+	/// Setting a new value to the property would call `startAnimating()` or
+	/// `stopAnimating()` depending on the value.
+	public var rac_animating: BindingTarget<Bool> {
```

Comment:
I think `rac` is debatable. We _might_ want binding target extensions of Foundation in ReactiveSwift too, which makes `rac` a bit less ideal. But `rac` is probably also the only choice anyway.

Or we may encourage operators instead, say `%view.isAnimating` as an equivalent of `view.reactive.isAnimating`